lib/indexer/links_lookup.rb

```diff
@@ -23,12 +24,19 @@ def prepare_tags(doc_hash)
       content_id = find_content_id(doc_hash)
       return doc_hash unless content_id
 
+      content_item = {
```

Comment:
Yeah, I was thinking the same thing. But I'm not sure about what to do about it yet.

At the moment, all the fields are optional, so 
if https://github.com/alphagov/govuk_taxonomy_helpers/pull/10 is merged, we could just call that method. This would give you a `LinkedContentItem` with a lot of null fields on it, but the `taxons` and `taxons_with_ancestors` methods would still work. This API still seems problematic though (having one big class with a lot of optional attributes/behaviour).